### ISSUE TITLE
npm audit reporting low level vulnerability from braces, transitive dependent of babel-cli

### ISSUE THREAD

**Issue by henrahmagix** [elastic/apm-agent-rum-js#283]
After installing the latest version available of this `4.0.2` I now get a failing `npm audit` and it cannot be auto fixed. Is it possible to fix here? Can `babel-cli` be removed as an installed dependency?

```
Low: Regular Expression Denial of Service
Package: braces
Patched in: >=2.3.1
Dependency of: @elastic/apm-rum
Path: @elastic/apm-rum > @elastic/apm-rum-core > babel-cli > chokidar > anymatch > micromatch > braces
More info: https://npmjs.com/advisories/786

Low: Regular Expression Denial of Service
Package: braces
Patched in: >=2.3.1
Dependency of: @elastic/apm-rum
Path: @elastic/apm-rum > babel-cli > chokidar > anymatch > micromatch > braces
More info: https://npmjs.com/advisories/786
```

Though I'm a bit confused, because I can't find `babel-cli` as any dependency in this repo:
https://github.com/elastic/apm-agent-rum-js/blob/%40elastic/apm-rum%404.0.2/packages/rum/package.json#L43-L46
https://github.com/elastic/apm-agent-rum-js/blob/%40elastic/apm-rum%404.0.2/packages/rum-core/package.json#L33-L38

What I can find is `@babel/cli`, but that's at v7, whilst I have 6.26.0 installed 😕 and a dev dependency at that, so it shouldn't be installed (right?)

Related: Babel have said that `babel-cli@6` is not maintained. See https://github.com/babel/babel/issues/9578#issuecomment-466922413

**Comment by henrahmagix** [comment]
Info from my `package-lock.json`:
```
    "@elastic/apm-rum": {
      "version": "4.0.2",
      "resolved": "https://registry.npmjs.org/@elastic/apm-rum/-/apm-rum-4.0.2.tgz",
      "integrity": "sha512-A0qfI0Pr/PomdHayRbp5pXUudW8ocbhxKOH4Z2DQVtZehoLfgng4bWYplwW+1BdeDSgwOoW67rTZi02u9HbyuQ==",
      "requires": {
        "@elastic/apm-rum-core": "^4.0.2",
        "babel-cli": "^6.26.0",
        "es6-promise": "^4.2.4"
      }
    },
    "@elastic/apm-rum-core": {
      "version": "4.0.2",
      "resolved": "https://registry.npmjs.org/@elastic/apm-rum-core/-/apm-rum-core-4.0.2.tgz",
      "integrity": "sha512-cejmDizG3NhFoDACEU+Y5jh0lkV6zUYSsmeoj5IBc2Ysb0JV5bacQDilKDiXDpAb+SGIDHNAV+5SPLLU68B5Wg==",
      "requires": {
        "babel-cli": "^6.26.0",
        "error-stack-parser": "^1.3.5",
        "opentracing": "^0.14.3",
        "stack-generator": "^1.0.7",
        "uuid": "^3.1.0"
      }
    },
```

**Comment by vigneshshanmugam** [comment]
@henrahmagix Thanks for reporting the issue, I have updated the dependencies to fix the low vulnerabilities.  #284

**Comment by henrahmagix** [comment]
Thanks very much @vigneshshanmugam!

**Comment by jahtalab** [comment]
We have released version `@elastic/apm-rum@4.1.1` which should resolve this issue. Please feel free to reopen if the issue persists.

**Comment by henrahmagix** [comment]
That's worked for me, thanks @jahtalab!
